.github/workflows/zizmor.yaml

```diff
@@ -0,0 +1,24 @@
+name: CI security linting
+
+on:
+  push:
+    branches: ["main"]
```

Comment:
How about only executing this workflow if anything was changed inside of `.github` using the [`path argument`](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-including-paths)? Or could we be missing something that way?

Reply:
Good idea, added accordingly.